### ISSUE TITLE
Accessibility : Types not specified for topics in search drop-down menu

### ISSUE THREAD

**Issue by sopu** [learningequality/ka-lite#3345]
Branch: Develop

The highlighted topic depicts a practice question set which is not obvious in first glance from the menu.
For screen readers such list items create a confusion as it cannot differentiate between the types of the listed items.

Screenshot:
![screenshot from 2015-03-19 16 11 49](https://cloud.githubusercontent.com/assets/6399612/6728799/b2297b2e-ce52-11e4-9f14-4246a3faadac.png)

Output from Fangs screen reader:
![screenshot from 2015-03-19 16 56 04](https://cloud.githubusercontent.com/assets/6399612/6729500/f56bc012-ce58-11e4-8e51-7e4250628cdb.png)

(No difference can be drawn from this output whether the topic is a practice question set or contains several subtopics)

Improvement: Just like quiz a "(practice)" could be appended with the topics which have practice question set to improve accessibility.

**Comment by MCGallaspy** [comment]
My first thought it simply to append the kind, so something like:

    My Cool Topic (Exercise)
    My Other Cool Topic (Video)

Any opinions? @radinamatic or others?

**Comment by MCGallaspy** [comment]
Specifically I'm wondering if there's a way to include the kind that shows up on screen readers only, given that the kind is already indicated by the icon in a non-screen-reader context.

**Comment by rtibbles** [comment]
Fixed in #3766
